packages/edgeconsent/README.md

```diff
@@ -1,4 +1,85 @@
 
-# React Native AEP Consent for Edge Network Extension
+# React Native Consent for Edge Network Extension
 
-// todo
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedgeconsent.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedgeconsent) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedgeconsent)](https://www.npmjs.com/package/@adobe/react-native-aepedgeconsent)
+
+`@adobe/react-native-aepedgeconsent` is a wrapper for the iOS and Android [Consent for Edge Network extension](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) to allow for integration with React Native applications.
+
+## Prerequisites
+
+The Consent for Edge Network extension has the following peer dependency, which must be installed prior to installing the Consent extension:
+- [Core](../core/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page. 
+
+Install the `@adobe/react-native-aepedgeconsent` package:
+
+```bash
+cd MyReactApp
+npm install @adobe/react-native-aepedgeconsent
+```
+
+## Usage
+
+### Installing and registering the extension with the AEP Mobile Core
+Install the Consent extension in your mobile property and configure the default consent preferences by following the steps in the [Consent for Edge Network extension documentation](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network).
+
+Then follow the same document for registering the Consent extension with the Mobile Core.
+Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
+
+### Importing the extension
+In your React Native application, import the AEPConsent extension as follows:
+```javascript
+import {AEPConsent} from '@adobe/react-native-aepedgeconsent';
+```
+
+## API reference
```

Comment:
also let me know what you think about having a separate API reference section/page. This example follows the GitBook doc format with syntax + example. 
cc @yangyansong-adbe